tests/test_lit_jupyter_component.py

```diff
@@ -1,12 +1,10 @@
-r"""
-To test a lightning component:
-
-1. Init the component.
-2. call .run()
-"""
+from unittest import mock
 from lit_jupyter.component import LitJupyter
 
 
-def test_placeholder_component():
-    component = LitJupyter()
-    # TODO: test that it's running
+@mock.patch("lit_jupyter.component.subprocess.Popen")
+def test_subprocess(mock_popen):
+    mock_popen().wait.return_value = 0
+    lightning_work = LitJupyter()
+    lightning_work.run()
+    return mock_popen.assert_called()
```

Comment:
@tchaton wdyt of this test?

Reply:
Yes, this is fine. But I would also check the arguments are correct.
This won't check that this component can be used properly locally and in the cloud. 
This is a mock test.